app/org/sagebionetworks/bridge/config/BridgeSpringConfig.java

```diff
@@ -101,6 +101,11 @@
         return Lists.newArrayList("REDISCLOUD_URL", "REDISTOGO_URL");
     }
     
+    @Bean(name = "newRedisProviders")
+    public List<String> newRedisProviders() {
+        return Lists.newArrayList(BridgeConfigFactory.getConfig().get("elasticache.url"));
+    }
```

Comment:
Cleaned this up, removing all the config rigamarole around falling back to Redis To Go.